src/os.ts

```diff
@@ -3,9 +3,13 @@ import * as os from 'os'
 export const isWindows10 =
 	process.platform === 'win32' && os.release().split('.')[0] === '10'
```

Comment:
This function returns true when OS is Windows 11. Please rename this function to 'isWindows10orUpper' or add exception case.